### DISCUSSION TITLE
Handlebar table template creation change

### DISCUSSION THREAD

**Discussion by ghost** [handlebars-lang/handlebars.js#1940] [Q&A]
JS fiddle [link](https://jsfiddle.net/p5ckx9wh/) of the issue.

I have a table where first row of the table each is defined. and in the second row tokens are defined, in the last row each declaration is done

While compiling two rows of each declaration and each ending declaration(ie first row and last row) is not removed.  Is there any way to remove those?

**Comment by jaylinski** [comment]
I'm not sure what HTML you want, but you can use `@last` or `@first` and do something like this:

```html
<script type="template/handlebars" id="template"><table style="border-collapse: collapse; width: 100%;" border="1">
<thead>
<tr>
<th>Name</th>
<th>Price</th>
<th>Quantity</th>
<th>Total</th>
</tr>
</thead>
<tbody>
{{#each itemList}}
<tr>
<td>{{name}}</td>
<td>{{price}}</td>
<td>{{qty}}</td>
<td>{{total}}</td>
</tr>
{{#unless @last}}
<tr>
<td colspan="4"></td>
</tr>
{{/unless}}
{{/each}}
</tbody>
</table></script>
```

**Reply by satyajitnayk** [reply]
@jaylinski is absolutely right. 
`@first` will check if the item is first element of iteration & likewise `@last`. https://handlebarsjs.com/api-reference/data-variables.html#first

But it would be great if you could provide screenshot or something as reference so that it can be understood what you want to achieve!
